extensions/amp-bind/0.1/amp-state.js

```diff
@@ -133,7 +133,7 @@ export class AmpState extends AMP.BaseElement {
     const id = user().assert(this.element.id, '<amp-state> must have an id.');
     const state = Object.create(null);
     state[id] = json;
-    bindForDoc(this.getAmpDoc()).then(bind => {
+    bindForDoc(this.element).then(bind => {
```

Comment:
This allows fetching the Bind service in the closest embed scope (e.g. in an AMP ad that uses Bind).